app/controllers/api/v1/short_form_controller.rb

```diff
@@ -3,14 +3,20 @@ class Api::V1::ShortFormController < ApiController
   before_action :authenticate_user!
 
   def submit
-    logger.debug "application_api_params: #{application_api_params}"
-    short_form_validator = ShortFormValidator.new(application_api_params)
-    if short_form_validator.valid?
-      application = application_service.submit(application_api_params)
-      logger.debug "application submit response: #{application}"
-      render json: { application: application }
+    # logger.debug "application_api_params: #{application_api_params}"
```

Comment:
@fddayan Our practice in the web app, which you haven't seen so I don't expect you to know it 🙂 has been to add a TODO comment whenever we add an error trigger for testing purposes. The format would be like: "TODO: Remove this testing error trigger after story acceptance. This is a trigger to [insert description of trigger here]." This helps us be totally clear and aware of what parts of a PR's code are for testing error triggers, and is an extra reminder to be sure to remove the error trigger before merge. Could you add a comment in that format here?